percpu/Cargo.toml

```diff
@@ -19,6 +19,7 @@ sp-naive = ["percpu_macros/sp-naive"]
 preempt = ["percpu_macros/preempt", "dep:kernel_guard"]
 
 default = []
+arm_el2 = []
```

Comment:
Change it to `arm-el2` for consistency with other features.

Reply:
resolved